book/src/development/adding_lints.md

```diff
@@ -116,40 +119,37 @@ Note that you can run multiple test files by specifying a comma separated list:
 
 ### Cargo lints
```

Comment:
At some point we should also add some documentation for method lints and the methods module. Just a reminder, having is book is already a big step and we can probably create an issue for someone to write additional documentation 🙃

Reply:
Agreed, I don't want to add too many new things in the PR though.